app/controllers/profile_controller.rb

```diff
@@ -9,6 +11,30 @@ def show
     @user = current_user
   end
 
+  def change_password
+    passwd_form = params[:password_change]
+    if passwd_form['new_password1'] != passwd_form['new_password2']
```

Comment:
once the form is a model, the validation should happen in there... however the prototype doesn't require typing the new password twice... @christopherthomasdesign can you confirm? the [Design System patterns](https://design-system.service.gov.uk/patterns/passwords/#hide-passwords-by-default) have:

>To help users meet your password constraints and prevent mistyped passwords, you can:
>...
> - make them enter their password twice and automatically compare them